README.md

```diff
@@ -1 +1,401 @@
+Endpoint: /clients/me/order-history
+Method: GET
+Request Body:
+```json
+{
+  
+}
+```
+14. Dish Ratings by Client
+Description: Allows a client to submit ratings for a dish.
+Endpoint: /clients/rating-dish/:id
+Method: POST
+Parameters:
+id (required): The ID of the dish to rate.
+Request Body:
+```json
+{
+  "rating": ""
+}
+```
+
+15. Get Chef Dishes by Client
+Description: Retrieves a list of dishes offered by a specific chef, based on a client's preferences.
+Endpoint: /clients/chefs/dishes/:id
+Method: GET
+Parameters:
+id (required): The ID of the chef to retrieve dishes from.
+
+16. Get Chef By Id
+Description: Retrieves information about a specific chef by their ID.
+Endpoint: /clients/chefs/:id
+Method: GET
+Parameters:
+id (required): The ID of the chef to retrieve.
+
+17. Create Chef Profile
+Description: Allows a user to create a chef profile to showcase their culinary offerings.
+Endpoint: /chef/
+Method: POST
+Request Body:
+```json
+{
+  
+    "restaurant": "",
+    "location": "",
+    "openingHours": "",
+    "closingHours": "",
+    "contactNumber": "",
+    "description": ""
+
+}
+```
+
+18. Edit Chef Profile
+Description: Allows the authenticated chef to edit their profile information.
+Endpoint: /chef/me
+Method: PUT
+Request Body:
+```json
+{
+  "restaurant": ""
+}
+```
+
+19. Retrieve Chef information
+Description: Retrieves the profile of a chef Chef information based on their ID.
+Endpoint: /chef/me
+Method: GET
+Parameters:
+id (required): The ID of the chef to retrieve.
+Request Body:
+```json
+{
+
+}
+```
+
+20. Retrieve Chef Dishes by chef
+Description: Retrieves a list of dishes offered by a certain chef.
+Endpoint: /chefs/:id/dishes
+Method: GET
+Parameters:
+id (required): The ID of the chef to retrieve dishes from.
+Request Body:
+```json
+{
+  
+}
+```
+
+21. Create Dish
+Description: Allows a chef to create a new dish and add it to their culinary offerings.
+Endpoint: /chef/dish
+Method: POST
+Request Body:
+```json
+{
+  
+    "name": "",
+    "description": "",
+    "price": "",
+    "rating": ""
+
+}
+```
+21. Get All Dishes
+Description: Retrieves all dishes offered by the chef.
+Endpoint: /chef/dish
+Method: GET
+
+22. Get Dish By Id
+Description: Retrieves a specific dish by its ID.
+Endpoint: /chef/dish/:id
+Method: GET
+Parameters:
+id (required): The ID of the dish to retrieve.
+
+23. Delete Dish By Id
+Description: Deletes a specific dish by its ID.
+Endpoint: /chef/dish/:id
+Method: DELETE
+Parameters:
+id (required): The ID of the dish to delete.
+
+24. Create Order
+Description: Creates a new order for the authenticated client.
+Endpoint: /orders
+Method: POST
+Request Body:
+```json
+{
+  
+}
+```
+25. Client Cancel Order
+Description: Allows a client to cancel an order they previously placed.
+Endpoint: /orders/cancel-order/:id
+Method: DELETE
+Parameters:
+id (required): The ID of the order to cancel.
+
+26. Get Chef Orders
+Description: Retrieves a list of orders received by a specific chef.
```

Comment:
lets indicate that this is only available to chefs
for example:
Allows a chef to retrieve the list of their orders